deploy/Chart/templates/rp/configmaps.yaml

```diff
@@ -13,17 +13,9 @@ data:
     environment:
       name: self-hosted
       roleLocation: "global"
-    storageProvider:
-      provider: "apiserver"
-      apiserver:
-        context: ""
-        namespace: "radius-system"
-    queueProvider:
-      provider: "apiserver"
+    storageProvider: {{ toYaml .Values.global.storageProvider | nindent 6 }}
```

Comment:
Same comments as above. Here we only need to change the namespace, not the whole block.

Reply:
However, I do like how DRY this is :)